Resources/Prototypes/Entities/Objects/Devices/Circuitboards/Machine/production.yml

```diff
@@ -940,6 +988,10 @@
       Capacitor: 4
     materialRequirements:
       Steel: 5
+  - type: ReverseEngineering # Nyano
+    difficulty: 3
```

Comment:
why is a bio fabricator the same intensity to research as this?

Reply:
I wanted to keep it at difficulty four must of went through it and messed it up